File: week3/Student.cpp
#include <iostream>
using namespace std;

class Student
{
  public:
    string fullName;
    int rollNum;
    double semPercentage;
    string collegeName;
    int collegeCode;
    Student()
    {
      cout<<"Student class initialized\n";
    }
    ~Student(){}
};

int main()
{
  Student Mounika;
  cout<<"Enter your name : ";
  getline(cin,Mounika.fullName);
  cout<<"Enter the roll number : ";
  cin>>Mounika.rollNum;
  cout<<"Enter the sem percentage : ";
  cin>>Mounika.semPercentage;
  cout<<"Enter the college name : ";
  cin>>Mounika.collegeName;
  cout<<"Enter college code : ";
  cin>>Mounika.collegeCode;
  cout<<"Name : "<<Mounika.fullName<<endl;
  cout<<"RollNum : "<<Mounika.rollNum<<endl;
  cout<<"SemPercentage : "<<Mounika.semPercentage<<endl;
  cout<<"CollegeName : "<<Mounika.collegeName<<endl;
  cout<<"CollegeCode : "<<Mounika.collegeCode<<endl;
  return 0;
}

File: week4/Student.java
import java.util.Scanner;

class Student13
{
  public
    String fullName;
    int rollNum;
    double semPercentage;
    String collegeName;
    int collegeCode;
    Student13()
    {
      System.out.print("Student class is created!!\nIam a student\n");
    }
    protected void finalize() throws Throwable
    {
      System.out.print("The class is finalized\n");
    }
}

public class Student
{
  public static void main(String[] args)
  {
    Student13 monika=new Student13();
    Scanner input=new Scanner(System.in);
    System.out.print("Enter your sweet name : ");
    monika.fullName=input.nextLine();
    System.out.print("Enter your rollNumber : ");
    monika.rollNum=input.nextInt();
    System.out.print("Enter your semPercentage : ");
    monika.semPercentage=input.nextDouble();
    input.nextLine();
    System.out.print("Enter your collegeName : ");
    monika.collegeName=input.nextLine();
    System.out.print("Enter your collegeCode : ");
    monika.collegeCode=input.nextInt();
    System.out.print("Name : "+monika.fullName+"\n");
    System.out.print("RollNumber : "+monika.rollNum+"\n");
    System.out.print("SemPercentage : "+monika.semPercentage+"\n");
    System.out.print("CollegeName : "+monika.collegeName+"\n");
    System.out.print("CollegeCode : "+monika.collegeCode+"\n");
    input.close();
  }
}
